environment_linux.yml

```diff
@@ -38,7 +38,4 @@ dependencies:
   - plotly=4.14.3
   - tabulate
   - pip:
-    - pydegensac
-
-
-
```

Comment:
Unrelated change

Reply:
Causes a mypy error... but I can add it back